src/components/pages/main-page/components/resource-card/ResourceCard.styles.ts

```diff
@@ -0,0 +1,50 @@
+import { SxProps } from '@mui/material/styles';
+import { Theme } from '@mui/system';
+
+export const card: SxProps<Theme> = {
+  width: {
+    mobile: '104px',
+    tablet: '200px',
+  },
+  height: {
+    mobile: '112px',
+    tablet: '200px',
+  },
+  background: 'rgba(38, 38, 38, 0.26)',
+  border: '1px solid #4A4A4A',
```

Comment:
Use color from palette. As I said, if front doesn't have colors from figma's palette you should add new color or change codes of colors.